build/changelog/build-changelog.php

```diff
@@ -4,8 +4,14 @@
 
 use Aws\Build\Changelog\ChangelogBuilder;
 
-$changelogBuilder = new ChangelogBuilder("", "", true);
+$params = [
+    'verbose_flag' => true
+];
+
+$changelogBuilder = new ChangelogBuilder($params);
```

Comment:
I think it's fine to make that the default behavior, but it seems odd not to expose this configurability to the command line.